modules/installation-osp-bootstrap-machine.adoc

```diff
@@ -16,10 +16,10 @@ The bootstrap machine requires:
 * A port from the {rh-openstack} quota
 * A flavor with at least 16 GB memory, 4 vCPUs, and 25 GB storage space
 
-[NOTE]
-====
-The installation program cannot pass certificate authority bundles to Ignition
-on control plane machines. Therefore, the bootstrap machine cannot retrieve
-Ignition configurations from Swift if your endpoint uses self-signed
-certificates.
-====
+// [NOTE]
+// ====
+// The installation program cannot pass certificate authority bundles to Ignition
+// on control plane machines. Therefore, the bootstrap machine cannot retrieve
+// Ignition configurations from Swift if your endpoint uses self-signed
+// certificates.
+// ====
```

Comment:
Installation on self-signed-certificate HTTPS is expected to work now

Reply:
Yeah, I commented that out assuming that it should work for this release, but didn't want to lose the text quite yet. :)